server/server.js

```diff
@@ -4,15 +4,21 @@ const cookieParser = require('cookie-parser');
 const app = express();
 const PORT = 3000;
 
+const userRouter = require('./routes/userRouter')
+const taskRouter = require('./routes/taskRouter')
+
 // including cookies, urlencoder, and express.json
 app.use(cookieParser());
 app.use(express.json());
 app.use(express.urlencoded());
 
 // if we need to serve any static files, serve them here:
-
+app.use(express.static(path.resolve(__dirname, '../client')));
```

Comment:
Please change to app.use('/client', express.static(path.resolve(__dirname, '../client/assets')));